galaxy/values.yaml

```diff
@@ -275,10 +294,18 @@ configs:
       database_connection: postgresql://unused:because@overridden_by_envvar
       integrated_tool_panel_config: "/galaxy/server/config/mutable/integrated_tool_panel.xml"
       sanitize_whitelist_file: "/galaxy/server/config/mutable/sanitize_whitelist.txt"
-      tool_config_file: "/galaxy/server/config/mutable/editable_shed_tool_conf.xml,/galaxy/server/config/tool_conf.xml"
+      tool_config_file: "/galaxy/server/config/tool_conf.xml{{if .Values.cvmfs.enabled}},{{.Values.cvmfs.main.mountPath}}/config/shed_tool_conf.xml{{end}}"
       shed_tool_config_file: "/galaxy/server/config/mutable/editable_shed_tool_conf.xml"
+      tool_data_table_config_path: |-
+        {{ if .Values.cvmfs.enabled -}}
+        {{ .Values.cvmfs.main.mountPath }}/config/shed_tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/managed/location/tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/byhand/location/tool_data_table_conf.xml
+        {{- end }}
       tool_dependency_dir: "{{.Values.persistence.mountPath}}/deps"
       job_config_file: "/galaxy/server/config/job_conf.xml"
+      builds_file_path: |-
+        {{ if .Values.cvmfs.enabled -}}
```

Comment:
Shouldn't this conditional be line above? ie, if not enabled, don't include `builds_file_path`.

Reply:
I don't think it can be the line above if we want to keep it valid yaml and not turn the whole thing into a multi-line string. If CVMFS is disabled, it will end up being an empty string which it seems galaxy just ignores and works as if it's not set.